src/test/java/de/uhd/ifi/se/decision/management/jira/persistence/decisiongroupmanager/TestInsertGroup.java

```diff
@@ -58,6 +62,28 @@ public void testInsertGroupElementNull() {
 	@NonTransactional
 	public void testInsertGroupArgsNotNull() {
 		assertTrue(DecisionGroupManager.insertGroup("TestGroup2", decisionKnowledgeElement) != -1);
+		assertTrue(DecisionGroupManager.getGroupsForElement(decisionKnowledgeElement).contains("TestGroup2"));
+	}
+
+	@Test
+	public void testInheritInsertGroup() {
+		new CodeFileExtractorAndMaintainer("TEST").extractAllChangedFiles();
+		KnowledgeGraph graph = KnowledgeGraph.getOrCreate("TEST");
+		List<KnowledgeElement> codeFiles = graph.getElements(KnowledgeType.CODE);
+
+		KnowledgeElement godClass = null;
+		for (KnowledgeElement codeFile : codeFiles) {

```

Comment:
In the future, we need to refactor the test code so that such methods are not duplicated/redundant.